# partMsg.msg
uint8 part_id
uint16 scene_id
actuator_lib/KondoServoMsg[] kondoServoSet
actuator_lib/BrushedMotorMsg[] brushedMotorSet
actuator_lib/BrushlessMotorMsg[] brushlessMotorSet
